lab-kaylyn/model/portfolio.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+const createError = require('http-errors');
+const debug = require('debug')('portfolio:portfolio');
+
+const Project = require('./project.js');
+
+const portfolioSchema = Schema({
+  name: {type: String, required: true},
+  about: {type: String, required: true},
+  projects: [{type: Schema.Types.ObjectId, ref: 'project'}],
+  work: {type: String, required: true},
+  timestamp: {type: Date, required: true},
+});
```

Comment:
@kaylynyuh schema looks good - nice use of the `ref` property to associate this with your project model